src/tests/yararules/testdata/regular_expressions.txt

```diff
@@ -1 +1 @@
-(({}))
+(((({6d64353a20}) & (({6f6e} | {6f6666}) & {73746174653a20}))))
```

Comment:
Since we have only one rule with regex, can you add one or two more tests? To make sure we don't introduce regressions in the future. The easiest way is probably

```
git clone https://github.com/Yara-Rules/rules/
cp $goodrules ./src/tests/yararules/testdata
python3 ./src/tests/yararules/generate_yaraparse_result_files.py $goodrules
```

Reply:
done